bitrise.yml

```diff
@@ -5,7 +5,7 @@ project_type: ios
 trigger_map:
   - push_branch: main
     pipeline: app-store-pipeline
-  - pull_request_target_branch: develop
+  - pull_request_target_branch: '*'
```

Comment:
curious - why *?

Reply:
@cyndichin without this tests will not run on feature branches. This will say make it so tests will run on every PR.

Reply:
thank you for the update!